backend/src/routes/api/notebooks/utils.ts

```diff
@@ -130,3 +138,41 @@ export const enableNotebook = async (
     }
   }
 };
+
+export const getElyraServiceAccountRoleBindingName = (notebookName: string): string =>
```

Comment:
You duplicated the logic? Why didn't you just move the logic? The frontend code is no longer used now.